falcon/request.py

```diff
@@ -220,6 +220,10 @@ def __init__(self, env, options=None):
         # Normalize path
         path = env['PATH_INFO']
         if path:
+            if six.PY3:
```

Comment:
Unfortunately, this needs a `# pragma no cover` comment since it is only covered for one of the test envs.